www/src/pages/foundations/typography.tsx

```diff
@@ -196,7 +196,7 @@ export default function TypographyPage({ pageContext }) {
             </tr>
             <tr>
               <td colSpan={2}>
-                <a href="#">Standalone Link</a>
+                <a href={location.pathname}>Standalone Link</a>
```

Comment:
This changes the link to refresh the page where the anchor didn't before. If the issue was an invalid anchor could we instead update these to be

```suggestion
                <a href="#links">Standalone Link</a>
```

so they go to the links heading they're under?

Reply:
Yes, these changes were added to resolve the error with the `jsx-a11y/anchor-is-valid` rule

```
227:17  error  The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid
```

I added `href="#links"` to all the links in the "Links" block.